starknet-foundry/crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -0,0 +1,187 @@
+use crate::helpers::constants::OZ_CLASS_HASH;
+use crate::helpers::scarb_utils::get_property;
+use anyhow::{anyhow, Context, Result};
+use camino::Utf8PathBuf;
+use cast::{get_network, parse_number, print_formatted};
+use clap::Args;
+use rand::rngs::OsRng;
+use rand::RngCore;
+use scarb::ops::find_manifest_path;
+use serde_json::json;
+use starknet::accounts::{AccountFactory, OpenZeppelinAccountFactory};
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_contract_address;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::JsonRpcClient;
+use starknet::signers::{LocalWallet, SigningKey};
+use std::fs::OpenOptions;
+use std::io::Write;
+use toml::Value;
+
+#[derive(Args, Debug)]
+#[command(about = "Create an account with all important secrets")]
+pub struct Create {
+    /// Account name under which secrets are going to be saved
```

Comment:
Technically, not only secrets. Maybe something like "... account information is going to be saved" ? or something similar?